CampaignConfig/mdc2020_corsika.cfg

```diff
@@ -1,23 +1,108 @@
 [global]
-includes = /mu2e/app/users/srsoleti/workDir/Production/CampaignConfig/mdc2020_prolog.cfg
-desc = MDC2020n
+includes = /cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020u/Production/CampaignConfig/mdc2020_prolog.cfg
+outdir_sim_nersc = /pnfs/mu2e/scratch/datasets/phy-sim/sim/mu2e/
```

Comment:
Given past experience with files disappearing from scratch we should ask if persistent is a better choice - if it is we need the workflow to include steps to clean up files from persistent after they are no longer needed.

Reply:
I consider this is a broader question outside the scope of this PR.